### PR TITLE
Basic cleanup of imports

### PR DESCRIPTION
**Pull request by troyraen**
This addresses the basic cleanup items in #96. Other items involving more structural changes are left for a future PR.

Changes to the notebook:

- remove all imports that are not actually used in the notebook, except those that need to be installed (currently in try/except blocks)
- switch unTimely to use `pip install`
- do a try/except to install `alerce`, so that it matches the others
- sort the imports

Changes to the code files/modules:

- add imports that were missing
- remove imports not currently used in the file
- sort the imports

Other changes:

- remove the empty unTimely directory

### CODE REVIEW

File: light_curves/multiband_lc.ipynb [hunk 1]
```diff
@@ -110,47 +91,39 @@
...
     "\n",
     "try:\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
     "except ImportError:\n",
-    "    if not os.path.exists('./unTimely_Catalog_explorer'):\n",
-    "        !git clone https://github.com/fkiwy/unTimely_Catalog_explorer.git\n",
-    "    sys.path.append('./unTimely_Catalog_explorer')\n",
+    "    !pip install git+https://github.com/fkiwy/unTimely_Catalog_explorer\n",
+    "    !pip install reproject\n",
     "    from unTimely_Catalog_tools import unTimelyCatalogExplorer\n",
```
Comment (bsipocz) [inline light_curves/multiband_lc.ipynb:105]: I suggest chopping up the imports and moving them to the beginning of the different big sections of the notebook. E.g. make it possible to obtain all the data without having any of the exploration tools installed/be ready.

File: light_curves/multiband_lc.ipynb [hunk 2]
```diff
@@ -110,47 +91,39 @@
...
-    "from plot_settings import setup_text_plots\n",
+    "\n",
     "\n",
     "## NEW! LOAD/SAVE PICKLE FILE! ##\n",
     "SAVEDF = False # if set to True, pickle file will be saved. If False, pickle file will be loaded.\n",
     "#pickle_file_name = \"data/dflc_Shooby_March102023.pkl\" # name of pickle file (to be loaded or saved)\n",
     "pickle_file_name = \"data/dflc_Krick_Feb232023.pkl\" # name of pickle file (to be loaded or saved)\n",
-    "###\n",
-    "\n"
+    "###"
```
Comment (bsipocz) [inline light_curves/multiband_lc.ipynb:126]: This should go when the notebook is ready to be shipped, but until then, please move to a separate cell.

**Comment by bsipocz** [comment]
The notebook is very nicely modular regarding the datasets, so I wonder whether it would be more useful to users if those cells would be stand-alone, too? E.g. move the imports into the cells, so it's copy pasteable as is. OTOH, people would need to have the custom functions, too, so maybe this is not as trivial.

File: light_curves/multiband_lc.ipynb (hunk 2 above)
Comment (troyraen) [reply light_curves/multiband_lc.ipynb:132]: thanks, done

**Comment by troyraen** [comment]
> move the imports into the cells, so it's copy pasteable as is

Happy to switch that but I'll let @jkrick weigh in first.

**Comment by jkrick** [comment]
@bsipocz @troyraen on the issue of whether or not we should move the more specialized imports further down into their respective cells: I think this brings up another issue.  The code currently is nicely modular with each archive, but assuming we get the multiprocessing working, we could theoretically just have two working cells.  One to get the data and a second to plot it.  However, it is nice for someone coming to the notebook to learn something, to have each archive call be a separate cell.  So, do we keep the serial calls to each archive for the point of it being a tutorial, or do we get rid of them?  And based on that answer, we could keep all imports at the top, or move them down.

**Comment by troyraen** [comment]
> it is nice for someone coming to the notebook to learn something, to have each archive call be a separate cell

To me, separate cells for the serial calls makes sense. And actually, either way, maybe those imports should stay at the top of the notebook because otherwise they should probably be duplicated in the serial and parallel sections, in case they are not executed in order.

**Review by jkrick — APPROVED** [review]
I just found this 'review changes' button that lets me officially say I have done a review.  Sorry for all the other times I missed this.  

Thanks for cleaning this all up, I think it looks good as is, and we can push the discussion of moving the imports down further into the cells to a later PR.

**Comment by jkrick** [comment]
I guess my question was if we should remove the serial calls altogether, or keep them.

**Comment by troyraen** [comment]
> I guess my question was if we should remove the serial calls altogether, or keep them.

Oh I see. In that case I think I still agree with this:

> it is nice for someone coming to the notebook to learn something, to have each archive call be a separate cell

It probably makes it easier to understand, and also easier because the user doesn't have to construct the code themselves if they want to try out a regular call without multiprocessing.

**Pull request merged**